core/src/main/java/org/bitcoinj/wallet/KeyChainGroupStructure.java

```diff
@@ -16,14 +16,54 @@
 
 package org.bitcoinj.wallet;
 
+import org.bitcoinj.core.NetworkParameters;
+import org.bitcoinj.crypto.ChildNumber;
 import org.bitcoinj.crypto.HDPath;
 import org.bitcoinj.script.Script;
 
 /** Defines a structure for hierarchical deterministic wallets. */
 public interface KeyChainGroupStructure {
+    ChildNumber PURPOSE_BIP44 = new ChildNumber(44, true);  // P2PKH
```

Comment:
Maybe we should (probably in another commit) also move the various constants from `DeterministicKeyChain` to here?

Reply:
Sounds good, I'll take a look.

Reply:
I actually think it might be better to put the ChildNumber constants in ChildNumber and the HDPath constants in HDPath.

Reply:
I moved the definitions of the various constants into `ChildNumber` and `HDPath`. What do you think of that? (See new force-push)

Note that `DeterministicKeyChain.BIP44_ACCOUNT_ZERO_PATH` is actually a MainNet path and the path for Testnet is different.

Reply:
Yes, those constants in `ChildNumber` and `HDPath` look much cleaner.